README.md

```diff
@@ -45,6 +46,7 @@ let props = {
   requiredCategory: ElementPropTypes.category,
   requiredCategory: ElementPropTypes.sectionHeader,
   requiredImage: ElementPropTypes.image,
+  requiredSlider: ElementPropTypes.slider,
```

Comment:
In this case, has the user the ability to specify any of these values in the SD config?:

```js
    min: 50,
    max: 100,
    stepSize: 5,
    selectedValue: 75
```

If so, maybe this prop is similar to "embeddable"

Reply:
`min`, `max`, and `stepSize` would be read-only in Site Designer, used as inputs for rendering the slider component. `selectedValue` is the only one that would be changed by the user.

Reply:
@dan-grayson Then, should we make it a shape to render that field in the SD, as, it's input? Showing or not in the SD will depend on the users specifying that field in the block config schema.

Reply:
@dan-grayson Or, are you showing by default that value to be edited?

Reply:
@santios-volusion yes, the selectedValue would be the default value displayed in the slider. We can have fallback behavior in Site Designer to ensure that we have something to display. Then when that's touched, it'll store in the correct schema.